runtime/tests/runtime_integration_tests/test_disable.rs

```diff
@@ -1,33 +1,137 @@
+    firmware::{APP_WITH_UART, FMC_WITH_UART},
+    ImageOptions,
+};
+use caliptra_common::mailbox_api::{CommandId, FwInfoResp, MailboxReqHeader, MailboxRespHeader};
 use caliptra_hw_model::HwModel;
+use dpe::{
+    commands::{CertifyKeyCmd, CertifyKeyFlags, Command, SignCmd, SignFlags},
+    context::ContextHandle,
+    response::Response,
+};
+use openssl::{
+    bn::BigNum,
+    ec::{EcGroup, EcKey},
+    ecdsa::EcdsaSig,
+    nid::Nid,
+};
 use zerocopy::{AsBytes, FromBytes};
 
-use crate::common::run_rt_test;
+use crate::common::{execute_dpe_cmd, run_rt_test, DpeResult, TEST_DIGEST, TEST_LABEL};
 
 #[test]
 fn test_disable_attestation_cmd() {
     let mut model = run_rt_test(None, None, None);
 
+    // sign the digest
+    let sign_cmd = SignCmd {
+        handle: ContextHandle::default(),
+        label: TEST_LABEL,
+        flags: SignFlags::empty(),
+        digest: TEST_DIGEST,
+    };
+    let resp = execute_dpe_cmd(&mut model, &mut Command::Sign(sign_cmd), DpeResult::Success);
+    let Some(Response::Sign(sign_resp)) = resp else {
+        panic!("Wrong response type!");
+    };
+
     let payload = MailboxReqHeader {
         chksum: caliptra_common::checksum::calc_checksum(
             u32::from(CommandId::DISABLE_ATTESTATION),
             &[],
         ),
     };
-    // once DPE APIs are enabled, ensure that the RT alias key in the cert is different from the key that signs DPE certs
     let resp = model
         .mailbox_execute(
             u32::from(CommandId::DISABLE_ATTESTATION),
             payload.as_bytes(),
         )
         .unwrap()
         .unwrap();
+    let resp_hdr = MailboxRespHeader::read_from(resp.as_bytes()).unwrap();
+    assert_eq!(
+        resp_hdr.fips_status,
+        MailboxRespHeader::FIPS_STATUS_APPROVED
+    );
 
+    // get pub key
+    let certify_key_cmd = CertifyKeyCmd {
+        handle: ContextHandle::default(),
+        label: TEST_LABEL,
+        flags: CertifyKeyFlags::empty(),
+        format: CertifyKeyCmd::FORMAT_X509,
+    };
+    let resp = execute_dpe_cmd(
+        &mut model,
+        &mut Command::CertifyKey(certify_key_cmd),
+        DpeResult::Success,
+    );
+    let Some(Response::CertifyKey(certify_key_resp)) = resp else {
+        panic!("Wrong response type!");
+    };
+
+    let sig = EcdsaSig::from_private_components(
+        BigNum::from_slice(&sign_resp.sig_r_or_hmac).unwrap(),
+        BigNum::from_slice(&sign_resp.sig_s).unwrap(),
+    )
+    .unwrap();
+    let ecc_pub_key = EcKey::from_public_key_affine_coordinates(
+        &EcGroup::from_curve_name(Nid::SECP384R1).unwrap(),
+        &BigNum::from_slice(&certify_key_resp.derived_pubkey_x).unwrap(),
+        &BigNum::from_slice(&certify_key_resp.derived_pubkey_y).unwrap(),
+    )
+    .unwrap();
+    // check that signature is unable to be verified by the pub key
+    assert!(!sig.verify(&TEST_DIGEST, &ecc_pub_key).unwrap());
+}
+
+#[test]
+fn test_attestation_disabled_flag_after_update_reset() {
```

Comment:
In this test we should also make sure that attestation still fails.

Reply:
Done